src/main/java/seedu/nutrigoals/logic/commands/FindCommand.java

```diff
@@ -2,41 +2,52 @@
 
 import static java.util.Objects.requireNonNull;
 
-import seedu.nutrigoals.commons.core.Messages;
+import java.util.Map;
+
+import seedu.nutrigoals.model.Calorie;
 import seedu.nutrigoals.model.Model;
-import seedu.nutrigoals.model.meal.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all food items in address book whose name contains any of the argument keywords.
- * Keyword matching is case insensitive.
+ * Finds and displays the calorie content of the specified {@code Food} (if any).
+ * Food name matching is case insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all food items whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds and displays the calorie content of the "
+            + "specified food\n"
+            + "Parameters: FOOD\n"
+            + "Example: " + COMMAND_WORD + " rice";
+
+    public static final String MESSAGE_FIND_FOOD_CALORIES_SUCCESS = "Calorie content of %1$s: %2$d kcal";
+    public static final String MESSAGE_FOOD_CALORIES_NOT_FOUND = "No calorie content information for %1$s";
 
-    private final NameContainsKeywordsPredicate predicate;
+    private final String foodName;
 
-    public FindCommand(NameContainsKeywordsPredicate predicate) {
-        this.predicate = predicate;
+    /**
+     * Creates a FindCommand to find the calorie content of the given {@code Food}.
+     * @param foodName The {@code Food} to find the calorie content of.
+     */
+    public FindCommand(String foodName) {
+        this.foodName = foodName.toLowerCase();
     }
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.updateFilteredFoodList(predicate.and(model.getDatePredicate()));
-        return new CommandResult(
-                String.format(Messages.MESSAGE_MEALS_LISTED_OVERVIEW, model.getFilteredFoodList().size()));
+        Map<String, Calorie> foodCalorieList = model.getFoodCaloriesList();
+        if (!foodCalorieList.containsKey(foodName)) {
+            return new CommandResult(String.format(MESSAGE_FOOD_CALORIES_NOT_FOUND, foodName));
+        }
+        int calories = foodCalorieList.get(foodName).getCalorieValue();
```

Comment:
Would it be better if we have method(s) in `Model` that does this computations, so that the internal details of `Model` isn't exposed?